liquidprompt

```diff
@@ -1419,10 +1421,16 @@ _lp_runtime()
         (( _LP_RUNTIME_SECONDS >= 60 )) && echo -n $((_LP_RUNTIME_SECONDS % 3600 / 60))m
         echo -n $((_LP_RUNTIME_SECONDS % 60))"s${NO_COL}"
     fi
+    if (( LP_ENABLE_RUNTIME_BELL && _LP_RUNTIME_SECONDS >= LP_RUNTIME_BELL_THRESHOLD ))
+    then
+        #bash reprints $PS1 on every SIGWINCH, so we tput to stderr here so the bell is not
+        #captured in PS1.  This way the bell only rings once, not on every terminal resize.
+        { tput bel || tput bl 2>/dev/null ; } >&2
```

Comment:
I was an idiot and put these redirections in the wrong order. That's why it stopped working right. See here for an explanation: http://mywiki.wooledge.org/BashFAQ/055
```suggestion
        { tput bel || tput bl ; } >&2 2>/dev/null
```

Reply:
Is there any reason to favor the redirect to devnull outside the {} vs leaving it inside?

Reply:
Tested and force pushed.